package.json

```diff
@@ -9,7 +9,7 @@
     "test": "npm run lint",
     "prepush": "npm test",
     "postinstall": "install-app-deps",
-    "pack": "npm run build && build --dir && babel --no-comments --compact --minified --out-file app/dist/bundle.js app/dist/bundle.js",
+    "pack": "npm run build && build --dir && BABEL_ENV=production babel --out-file app/dist/bundle.js --no-comments --minified app/dist/bundle.js",
```

Comment:
I think people just use cross-env for that - the env option just reads from process.env - otherwise you would just make your babelrc a js file not in package.json and do whatever you want from there?